pkg/commands/compute/publish.go

```diff
@@ -21,6 +21,8 @@ type PublishCommand struct {
 	domain         cmd.OptionalString
 	backend        cmd.OptionalString
 	backendPort    cmd.OptionalUint
+	overrideHost   cmd.OptionalString
+	sslSNIHostname cmd.OptionalString
```

Comment:
NIT: I'd prefer to order these Deploy fields.

Reply:
Done.